frontend/public/locales/fr/personal-information.json

```diff
@@ -115,8 +154,7 @@
         "error": "(FR) Error"
       },
       "error-message": {
-        "empty-phone-number": "(FR) Enter a phone number",        
-        "invalid-phone-format":"(FR) Invalid phone number"
+        "empty-phone-number": "(FR) Enter a phone number"
```

Comment:
`invalid-phone-format` should remain as it is still being referenced.

Reply:
added back